### PR TITLE
Update azure entities

### PR DESCRIPTION
**Pull request by areina**
### Relevant information

Update a bunch of Entity Types coming from Azure integrations. I updated golden tags, summary metrics and golden metrics.

### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [x] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by IreneMLC — APPROVED** [review]

File: definitions/infra-azurecosmosdbcollection/summary_metrics.yml [hunk 1]
```diff
@@ -3,17 +3,24 @@ providerAccountName:
     key: providerAccountName
   title: Azure Account
   unit: STRING
-totalRequests:
+requests:
```
Comment (naxhh) [inline definitions/infra-azurecosmosdbcollection/summary_metrics.yml:6]: based on the other files. are the metrics here missing the `facet: entityName` ?

**Review by naxhh — COMMENTED** [review]
nit comment about the entity name field in some metrics

File: definitions/infra-azurecosmosdbcollection/summary_metrics.yml (hunk 1 above)
Comment (areina) [reply definitions/infra-azurecosmosdbcollection/summary_metrics.yml:6]: If I'm not wrong, `facet: entityName` has to be specified on golden metrics files, right?

Comment (naxhh) [reply definitions/infra-azurecosmosdbcollection/summary_metrics.yml:6]: totally true. my bad!

**Review by naxhh — APPROVED** [review]

**Pull request merged**